requirements.txt

```diff
@@ -5,6 +5,7 @@ Flask-WTF>=0.13.1
 Flask-Login
 Flask-Mail
 psycopg2>=2.6.2
+selenium
```

Comment:
Sure makes sense. To the best of my knowledge that's how it's usually handled, so in commit 8043c0b I've split the requirements into two files, one that contains just the requirements for staging/prod (`requirements.txt`) and then another file `dev-requirements.txt` that contains the additional requirements required for dev/testing. We will also want to modify the development VM environment to have it install both files